src/adapters/postgres/cohortMembers-adapter.ts

```
@@ -2191,6 +2191,7 @@ export class PostgresCohortMembersService {
         'parentId',
         'type',
         'status',
+        'metadata',
```

ESLint

[error] Replace `'metadata'` with `"metadata"`

(prettier/prettier)